docs/source/tutorial_metrics.md

```diff
@@ -435,12 +435,12 @@ please [file an issue on GitHub](https://github.com/facebookresearch/ParlAI/issu
 | `interdistinct-1`, `interdictinct-2` | Fraction of n-grams unique across _all_ generations |
 | `intradistinct-1`, `intradictinct-2` | Fraction of n-grams unique _within_ each utterance |
 | `jga`                   | Joint Goal Accuracy |
-|`label_length`           | Average length of label tokens per batch |
-|`label_truncate`         | Ratio of samples' labels being truncated per batch |
+| `llen`                  | Average length of label in number of tokens |
 | `loss`                  | Loss |
 | `lr`                    | The most recent learning rate applied |
 | `ltpb`                  | Label tokens per batch |
 | `ltps`                  | Label tokens per second |
+| `ltrun`                 | Fraction of samples with some label truncation |
```

Comment:
ditto. totally fine to leave it if we want to keep abbreviations 5 chars or less